dialyzer.ignore-warnings

```diff
@@ -99,3 +99,8 @@ Unknown functions:
   rt:wait_until_transfers_complete/1
   yz_solr_orig:index_batch_orig/2
   yz_solrq_drain_mgr_orig:drain_orig/1
+  yz_solrq_drain_mgr_orig:unlink_and_kill_orig/2
+  yz_solrq_drain_fsm_orig:prepare_orig/2
+  yz_solrq_drain_mgr_orig:unlink_and_kill_orig/2
+  yokozuna_rt:commit/2
```

Comment:
probably don't need these as yokozuna_rt refers to the r_t proper (repo) helper mod.